mantle/kola/tests/misc/boot-mirror.go

```diff
@@ -100,6 +100,11 @@ func runBootMirrorTest(c cluster.TestCluster) {
 			MinMemory:       4096,
 		},
 	}
+	// ppc64le and aarch64 use 64K pages
```

Comment:
Hmm, good point. For reference, I cargo-culted this block from other places in the code  for the ppc64le bit but reading the comment it's true that its mention of aarch64 using 64K pages is incorrect (hence why we're doing things like adding support for a separate aarch64 kernel [with the feature](https://github.com/openshift/machine-config-operator/pull/3903)).

It looks like the first comment mentioning this was introduced in https://github.com/coreos/coreos-assembler/pull/1955. The comment may not be accurate but it seems like Prashanth had to bump the memory on aarch64 too to get tests to pass.

For now, I've cross-referenced all these occurrences, but let's look into ripping them out again to see which ones are load-bearing and only keep those (with an up-to-date rationale).

Reply:
aarch64 used 64k pages by default with RHEL 8. RHEL 9 defaulted to 4K pages, that's why we're introducing #3903
Maybe that's why it was there ?

Reply:
I'm still not super comfortable with the comment as is because it can be misleading. i.e. someone reads the comment and accepts it as fact that aarch64 uses 64K pages.. how about we add a `(sometimes)` after `aarch64`:

```
// ppc64le and aarch64 (sometimes) use 64K pages
```

Maybe we even add the `(sometimes)` in a second commit so the commit message can have more context.

Reply:
> aarch64 used 64k pages by default with RHEL 8. RHEL 9 defaulted to 4K pages, that's why we're introducing #3903 Maybe that's why it was there ?

Nice find! Yes, it all makes sense now.

> Maybe we even add the (sometimes) in a second commit so the commit message can have more context.

Hmm, I think we should actually drop the aarch64 special-casing entirely. I'm working on a patch right now and will test it. Assuming nothing breaks, I'll push it as another commit here.

Reply:
OK, I ended up splitting this out into a separate PR to not waste CI here: https://github.com/coreos/coreos-assembler/pull/3617.